utils/html.go

```diff
@@ -86,6 +86,14 @@ func (t *HTMLTemplate) addDefaultProps() {
 	}
 
 	t.Props["Footer"] = localT("api.templates.email_footer")
+	t.Props["Organization"] = localT("api.templates.email_organization")
```

Comment:
This would be less confusing if you did the localT call down where your constructing the string, instead of appending.